--- a/drivers/input/oplus_fp_drivers/oplus_fp_common/oplus_fp_common.c
+++ b/drivers/input/oplus_fp_drivers/oplus_fp_common/oplus_fp_common.c
@@ -25,7 +25,7 @@
 #include <linux/platform_device.h>
 #include <linux/device.h>
 #include <linux/mutex.h>
-#include <soc/qcom/smem.h>
+#include <linux/soc/qcom/smem.h>
 #include <linux/fb.h>
 #include <linux/notifier.h>
 #include "../include/oplus_fp_common.h"
